pageserve/app.py

```diff
@@ -22,24 +22,93 @@
 
 app = Flask(__name__)
 
-@app.route('/')
+
+@app.route('/v1/')
 def index():
     user = get_user()
-    is_auth = get_auth(user)
+    is_auth = has_edit_access(get_auth_json(user))
+    posts = get_posts()
     return render_template(
         'index.html',
+        posts=posts,
         auth=is_auth,
     )
-    
-def get_auth(user):
+
+
+@app.route('/v1/events')
+def show_events():
+    user = get_user()
+    is_auth = has_edit_access(get_auth_json(user))
+    events = get_events()
+    return render_template(
+        'events.html',
+        events=events,
+        auth=is_auth,
+    )
+
+
+def get_posts():
+    # TODO: integrate with posts service to pull post info from database
+    posts = [{'post_id': '1',
+              'event_id': '0',
+              'type': 'text',
+              'author': 'admin',
+              'created_at': '7-9-2019',
+              'text': 'this will be a fun event!',
+              },
```

Comment:
Do you want to handle images yet or implement that later?

Reply:
I'll still take images from the posts service but if I don't get to displaying them I'll just put a placeholder instead